NASA-AMMOS/aerie-ui
# Expose model upload errors

Resolves #1248 

To test:

1. Start up local development as usual
2. Navigate to model upload page
3. Stop the `aerie_merlin` container
4. Proceed with creating a model
5. Verify that it forwards you to the newly created model
6. Verify that errors appear on the created model's page
7. Create a plan using the broken model
8. Verify that a banner appears at the top stating that model errors are present
9. Click on the "View Errors in Console" button
10. Verify that the console opens up and displays the model errors with clickable buttons to view more information about each error

Follow-up ticket: https://github.com/NASA-AMMOS/aerie-ui/issues/1326
